Sources/SWBCore/DependencyResolution.swift

```diff
@@ -528,11 +541,18 @@ extension SpecializationParameters {
                     let buildParams = buildRequest.parameters.replacing(activeRunDestination: runDestination, activeArchitecture: archName)
                     let specializationParams = SpecializationParameters.default(workspaceContext: workspaceContext, buildRequestContext: buildRequestContext, parameters: buildParams)
                     platformBuildParameters.append(PlatformBuildParameters(buildParams: buildParams, specializationParams: specializationParams))
+
+                    if buildParams.activeRunDestination?.platform == hostOS && sdkVariant == matchingSDK.defaultVariant?.name  {
```

Comment:
Not sure if it is just me, but this tripped me up when reading because it seemed like we're using disjunct things for the comparison. Can we just get the SDK variant via the run destination as well?

Reply:
Sure